.github/workflows/main.yml

```diff
@@ -19,7 +19,8 @@ jobs:
     - name: Download zig
       run: |
         export PYTHONIOENCODING=utf8
-        wget $(curl -s 'https://ziglang.org/download/index.json' | python3 -c "import sys, json; print(json.load(sys.stdin)['master']['x86_64-linux']['tarball'])")
+        # Lock the master commit that we download, because of blocking issues
+        wget https://ziglang.org/builds/zig-linux-x86_64-0.8.0-dev.2133+ad33e3483.tar.xz #$(curl -s 'https://ziglang.org/download/index.json' | python3 -c "import sys, json; print(json.load(sys.stdin)['master']['x86_64-linux']['tarball'])")
```

Comment:
Could remove the commented previous version as it is in the commit history if we need it.

Reply:
Roger roger

Reply:
Donezo